docs/home/howto/CLUSTER.md

```diff
@@ -179,11 +181,13 @@ To setup the cluster do the following steps from the provisioning machine:
 
 ## How to create an Epiphany cluster on existing airgapped infrastructure
 
+*Please read first prerequisites related to [hostnames requirements](./PREREQUISITES.md#hostnames-requirements).*
+
 Epicli has the ability to setup a cluster on airgapped infrastructure provided by you. These can be either bare metal machines or VM's and should meet the following requirements:
 
 *Note. Hardware requirements are not listed since this dependends on use-case, component configuration etc.*
 
-1. The airgapped cluster machines/vm`s are connected by a network or virtual network of some sorts and can communicate which each other:
+1. The airgapped cluster machines/vm's are connected by a network or virtual network of some sorts and can communicate which each other.
```

Comment:
If you change `'` here, why it's not changed in other places?

Reply:
Will do this later on.